automation/vars/hci.yaml

```diff
@@ -76,3 +76,78 @@ vas:
           - name: edpm-deployment-values-post-ceph
             src_file: values.yaml
         build_output: deployment-post-ceph.yaml
+
+# Adoption related content for adoption
+adoption:
+  hci:
+    layout_patch:
+      vms:
+        # Let's remove the default computes, since we want to adopt the
+        # OSP ones
+        compute:
+          amount: 0
+        osp-undercloud:
+          amount: 1
+          disk_file_name: rhel-8.10.qcow2
```

Comment:
I'm wondering if we want this level of detail in architecture, where we define a bunch of VMs settings.
I would have expected to have this kind of definition in ci-framework (like we do for greenfield, right?) and reference a "scenario" that can be loaded.
However, I'm not saying this is a bad approach, as we have a clear view of the infra associated to VA1 in the adoption context in architectures.
I'll let @marios @jistr @cescgina comment further on this.

Reply:
we can. It could point to a thing to inherit from, but there are some caveats there, like the file location (remote vs ansible-controller). Indeed, this might be linked a bit too much to the framework, while having a description file could do.

And if we go down that path, we can push everything in that file - leading to this automation, in architecture, to have "just" `adoption: relative/path/to/scenario.yml`
But there would still be the "where to store t-h-t static/dynamic content that needs to be injected"... And there, I'm sure it doesn't belong to the ci-framework. Probably "bad" for architecture. But I'm not sure if there would be any other "good" place :/

Reply:
I think that a VA/DT already defines a set of services and it can be used as source of truth to define a static set of THT env files. That part is not "strictly" connected to the infra IMO, and it might live here in terms of "description" of the source cloud.

WRT the infra part, we could minimize the changes here by referencing a scenario that lives in the `adoption` ci-framework path, but I'd like to hear more opinions.
My comment started from the assumption that for greenfield VA/DTs we have scenario files that live in ci-framework,  and we might want to be consistent with the already existing approach.

Reply:
What if this infra style change to hci.yaml were in `ci-framework/scenarios/adoption/hci.yaml`?

Then in `architecture/automation/vars/hci.yaml` it could have a reference to `ci-framework/scenarios/adoption/hci.yaml`?

Reply:
we could, indeed. We'd even NOT need any reference from any file: the scenario name, and the fact it's "adoption" should be sufficient to built the pointer to the file.

It won't solve the "where do we put OSP t-h-t content", but it reduces the amount of affected repositories in case of changes. I didn't think about it, and I'm ashamed of that ;_;.

Far, far cleaner to pass via scenario files - and that's the purpose if them anyway!